includes/utilities.inc

```diff
@@ -205,3 +205,70 @@ function islandora_batch_scan_check_encoding($type, $encoding) {
     }
   }
 }
+
+/**
+ * Batch to delete objects ingested via a batch set.
+ *
+ * @param int $set_id
+ *   Batch set ID.
+ */
+function islandora_batch_batch_delete_ingested_object_in_set($set_id) {
+  module_load_include('inc', 'islandora_batch', 'includes/db');
+  $batch = array(
+    'title' => t('Delete ingested objects from set'),
+    'finished' => 'islandora_batch_batch_delete_ingested_object_in_set_finished',
+    'init_message' => t('Initializing...'),
+    'operations' => array(),
+    'progress_message' => t('Time elapsed: @elapsed.'),
+    'file' => drupal_get_path('module', 'islandora_batch') . '/includes/utilities.inc',
```

Comment:
I feel like this could all merit its own file... `includes/set-delete.batch.inc` or something? I'm terrible with names.